python/cudf/cudf/core/indexed_frame.py

```diff
@@ -2174,13 +2174,15 @@ def bfill(self, value=None, axis=None, inplace=None, limit=None):
         -------
             Object with missing values filled or None if ``inplace=True``.
         """
-        return self.fillna(
-            method="bfill",
-            value=value,
-            axis=axis,
-            inplace=inplace,
-            limit=limit,
-        )
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
```

Comment:
Nit: Could we add `FutureWarning` to the filter?

Reply:
Done 👍